table.scss

```diff
@@ -0,0 +1,126 @@
+@import 'bower_components/vui-colors/colors.scss';
+@import 'bower_components/vui-typography/typography.scss';
+@import 'bower_components/vui-typography/px-to-base-rem.scss';
+
+@mixin vui-table() {
+	background-color: #ffffff;
+	border: 1px solid $vui-color-titanius;
+	border-collapse: separate;
+	border-radius: 6px;
+	border-spacing: 0;
+	display: table;
+	width: 100%;
+
+	& thead {
+		display: table-header-group;
+
+		& th {
+			@include vui-table-header-cell();
+		}
+	}
+
+	& tfoot {
+		& tr:first-child td {
+			border-top: 1px solid $vui-color-titanius;
+		}
+		& tr:first-child th {
+			border-top: 1px solid $vui-color-titanius;
+		}
+		display: table-footer-group;
+	}
+
+	& tr {
+		display: table-row;
+	}
+
+	& td {
+		@include vui-table-cell();
+	}
+
+	& tr td:last-child {
+		border-right: none;
+	}
+
+	& tr:last-child td {
+		border-bottom: none;
+	}
+
+	& tbody th,
+	  tfoot th {
+		@include vui-table-cell();
+	}
+
+	& tr th:last-child {
+		border-right: none;
+	}
+
+	//Correct for corners bleeding outside the rounded corners
+	& thead > tr:first-child > :first-child,
+	  tbody:only-child > tr:first-child > :first-child,
+	  tfoot:first-child + tbody:last-child > tr:first-child > :first-child,
+	  caption:first-child + tbody:last-child > tr:first-child > :first-child,
+	  caption:first-child + tfoot + tbody:last-child > tr:first-child > :first-child,
+	  caption:first-child + colgroup + tbody:last-child > tr:first-child > :first-child,
+	  caption:first-child + colgroup + tfoot + tbody:last-child > tr:first-child > :first-child,
+	  colgroup:first-child + tbody:last-child > tr:first-child > :first-child,
+	  colgroup:first-child + tfoot + tbody:last-child > tr:first-child > :first-child {
+		border-top-left-radius: 6px;
```

Comment:
Looks like RTL would be an issue for this as well... at first I thought because it is a table the browser might handle it, but that doesn't look like the case.